model/src/main/kotlin/config/RepositoryConfiguration.kt

```diff
@@ -57,7 +57,9 @@ private class ExcludesFilter {
 @Suppress("EqualsOrHashCode", "EqualsWithHashCodeExist") // The class is not supposed to be used with hashing.
 private class ResolutionsFilter {
     override fun equals(other: Any?): Boolean =
-        if (other is Resolutions) other.issues.isEmpty() && other.ruleViolations.isEmpty() else false
+        if (other is Resolutions) {
+            other.issues.isEmpty() && other.ruleViolations.isEmpty() && other.vulnerabilities.isEmpty()
+        } else false
 }
```

Comment:
other is Resolutions && other.issues.isEmpty() && other.ruleViolations.isEmpty() && other.vulnerabilities.isEmpty() ?